Dockerfile

```diff
@@ -55,13 +55,14 @@ RUN mkdir -p /tmp/rpms && \
 RUN pip3 install --user botocore
 
 # This image is equivalent to the eks-distro-minimal-base-python image but with pip installed as well
-FROM public.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-python-builder:3.9-al2 as rpm-installer
+FROM public.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-python-builder:3.9.16-al23 as rpm-installer
 
 COPY --from=rpm-provider /tmp/rpms/* /tmp/download/
 
 # second param indicates to skip installing dependency rpms, these will be installed manually
 # cd, ls, cat, vim, tcpdump, are for debugging
 RUN clean_install amazon-efs-utils true && \
+    clean_install crypto-policies true && \
```

Comment:
Required to avoid `/etc/crypto-policies/back-ends/opensslcnf.config` not found error from openssl